luda/editor/client/src/sequenceListView/operations/loadSequence.ts

```diff
@@ -0,0 +1,47 @@
+import fileSystem from "../../fileSystem/fileSystem";
+import { TimelineState, Track } from "../../timeline/type";
+import { SequenceListViewState } from "../type";
+
+export async function loadSequence(
```

Comment:
I think `loadSequence` won't resume if user save state and restore during loading.
I know it's not that much matter but I want to support namui's application works even though it serialize state and deserialize later.
Let me think how to support this in `loadSequence`. I think it should be like `ImageLoadManager`.

Reply:
Fixed with 6b75133